README.md

```diff
@@ -46,6 +46,52 @@ and are automatically released (multiple times a day, when needed) by
 Type checkers should be able to use these stub packages when installed. For more
 details, see the documentation for your type checker.
 
+### Package versioning for third-party stubs
+
+Version numbers of third-party stub packages consist of at least four parts.
+All parts of the stub version, except for the last part, correspond to the
+version of the runtime package being stubbed. For example, if the `types-foo`
+package has version `1.1.2.0`, this signifies that the package contains stubs
+for `foo==1.1.2`. In this example, the final element of the version
+indicates that this is the first revision of the stubs for `foo`. If an update
+to the stubs were pushed (but the stubs were still aiming to provide
+annotations for `foo==1.1.2`), then the version of `types-foo` would
+increment to `1.1.2.1`.
+
+At typeshed, we try to keep breaking changes to a minimum. However, due to the
+nature of stubs, any version bump can introduce changes that might make your
+code fail to type check.
+
+There are several strategies available for specifying the version of a stubs
+package you're using, each with its own tradeoffs
```

Comment:
```suggestion
package you're using, each with its own tradeoffs:
```

Reply:
duplicate of https://github.com/python/typeshed/pull/10840#discussion_r1347533070 ;)

Reply:
I spent too much time convincing myself that the "at least four parts" is right :) (Seems like pywin32 itself uses a two-part versioning scheme, but we add enough .0s to indeed make it four-part.)